.docker/php-fpm/7.1/config/entrypoint.sh

```diff
@@ -7,18 +7,25 @@ shopt -s nocasematch
 if [ -z ${TZ+x} ]; then
   echo "Warning : TimeZone is unset !";
 else
-  ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone ;
+  sudo ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ | sudo tee /etc/timezone ;
 fi
 
+if [ $XDEBUG_ENABLE == "true" ]; then
+  sudo sed -i "s#XDEBUG_REMOTE_HOST#$(ip r | awk '/default/{print $3}')#" /usr/local/etc/php/conf.d/xdebug.ini
+else
+  echo "XDebug disabled";
+fi
+
+if [ $REMOTE_HOST_ENABLE == "true" ]; then
```

Comment:
Why configuring this variable if xdebug is disabled ?

Reply:
It's in case you need the DNS for the host for something else (no more need for the external_host with the ip of the host, you can use the docker internal network)

Reply:
It's in case you need the DNS for the host for something else (no more need for the external_host with the ip of the host, you can use the docker internal network)